walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -58,8 +55,19 @@ public void setUp() {
     @Test
```

Comment:
Maybe add a description to make it clear what you are testing

Reply:
the testing function name is alr "targetFunction_inputDescription_expectedOutcome"; i consulted Thach that no need javadoc for it